### DISCUSSION TITLE
Hiding Navigation Bar Missing

### DISCUSSION THREAD

**Discussion by spitfire94** [Mahmud0808/Iconify#120] [General]
I use [FNG](https://play.google.com/store/apps/details?id=com.fb.fluid) to replace the nav-bar and gestures. Originally used Substratum to disable the aosp nav-bar entirely, but chose to use this app when finding the same functionality. The most recent update seems to have removed this ability; I'd like it restored.

**Comment by Mahmud0808** [comment]
![Screenshot_20230402-080709_Iconify](https://user-images.githubusercontent.com/29881338/229327164-89d9ddc3-6734-4075-86a8-4c90c4f30c95.png)

Are you talking about these options? It's still there, not removed.
